packages/desktop/Cargo.toml

```diff
@@ -29,13 +29,18 @@ sysinfo = "0.30"
 tokio = { version = "1.33", features = ["full"] }
 tracing = "0.1"
 tracing-subscriber = { version = "0.3", features = ["env-filter"] }
+netdev = "0.24"
+regex = "1"
 
 [target.'cfg(all(target_os = "windows", target_arch = "x86_64"))'.dependencies]
 komorebi-client = { git = "https://github.com/LGUG2Z/komorebi", rev = "1a8b6a7398b12b3686b80c6228875a79e7165a26" }
 
 [target.'cfg(target_os = "macos")'.dependencies]
 cocoa = "0.25"
 
+[target.'cfg(windows)'.dependencies]
+winapi = {version = "0.3.9", features = ["winuser", "processthreadsapi", "handleapi", "psapi"] }
```

Comment:
could we use `windows-rs` instead? `winapi` hasn't been maintained in a while, and the `windows-rs` one is actively maintained by msoft

Reply:
winapi was used as a dependency for giving a "DETACHED_PROCESS" flag when running the netsh command which I copied from anther project

I tried to find a 1:1 copy of it in windows-rs but wasn't successful, it might be out there in some form

in any case, it seems the flag isn't actually needed, my dev environment is running fine without so I've deleted the dependency